src/encoded/renderers.py

```diff
@@ -4,35 +4,33 @@
 import psutil
 import time
 
-from pkg_resources import resource_filename
-from urllib.parse import urlencode
+from dcicutils.misc_utils import environ_bool, PRINT, ignored
 from functools import lru_cache
+from pkg_resources import resource_filename
 from pyramid.events import BeforeRender, subscriber
 from pyramid.httpexceptions import (
     HTTPMovedPermanently,
     HTTPPreconditionFailed,
     HTTPUnauthorized,
-    # HTTPForbidden,
     HTTPUnsupportedMediaType,
     HTTPNotAcceptable,
     HTTPServerError
 )
-# from pyramid.security import forget
+from pyramid.response import Response
 from pyramid.settings import asbool
 from pyramid.threadlocal import manager
-from pyramid.response import Response
 from pyramid.traversal import split_path_info, _join_path_tuple
-# from snovault.validation import CSRFTokenError
-# from subprocess_middleware.tween import SubprocessTween
 from subprocess_middleware.worker import TransformWorker
+from urllib.parse import urlencode
 from webob.cookies import Cookie
+from .util import content_type_allowed
 
 
 log = logging.getLogger(__name__)
 
 
 def includeme(config):
-    '''
+    """
```

Comment:
A lot of the changes in this file are PEP8, things like fixing doc strings to use triple double quotes rather than triple single quotes, breaking long lines, etc. I've added comments where there are semantic changes.